bftengine/tests/bcstatetransfer/test_app_state.hpp

```diff
@@ -77,16 +81,17 @@ class TestAppState : public IAppState {
     memcpy(&bl.block, block, blockSize);
     bl.actualSize = blockSize;
     last_block_ = blockId;
+    blocks_.emplace(blockId, bl);
```

Comment:
so what did it do until now?
the block was not really put anywhere?

Reply:
Yes, block was not written anywhere.